### PR TITLE
Fixes #8231, Plasmamen being forevially jittery, drowsy, confused, etc etc etc fuck whoever did this

### LINKED ISSUE #8231: Plasmamen never ever stop jittering
Their jitteriness var is always set to 30 and never goes down even after being warmed up.

### PR DESCRIPTION
**Pull request by ghost**
This also makes plasmamen finally feel hunger. Bugfix?

### CODE REVIEW

File: __DEFINES/setup.dm [hunk 1]
```diff
@@ -967,12 +967,13 @@ var/list/RESTRICTED_CAMERA_NETWORKS = list( //Those networks can only be accesse
...
+#define PLASMA_IMMUNE 8192
 
-#define CAN_BE_FAT 8192 // /vg/
+#define CAN_BE_FAT 16384 // /vg/
 
-#define IS_BULKY 16384 //can't wear exosuits, gloves, masks, or hardsuits
+#define IS_BULKY 32768 //can't wear exosuits, gloves, masks, or hardsuits
 
-#define NO_SKIN 32768
+#define NO_SKIN 65536 //Is there a limit to how high flag defines can go?
```
Comment (9600bauds) [inline __DEFINES/setup.dm]: important, please advise

Comment (ghost) [reply __DEFINES/setup.dm]: Yes

Comment (9600bauds) [reply __DEFINES/setup.dm]: yes as in "this won't work"?

Comment (9600bauds) [reply __DEFINES/setup.dm]: okay yeah I just tested and it indeed doesn't work

I'll probably just replace the IS_WHITELISTED flag

Comment (Intigracy) [reply __DEFINES/setup.dm]: 2^0 through 2^15

Comment (9600bauds) [reply __DEFINES/setup.dm]: so do I just cannibalize another flag or add a proc for this

File: code/modules/mob/living/carbon/human/life/handle_chemicals_in_body.dm [hunk 2]
```diff
@@ -22,14 +22,13 @@
 
 	var/total_plasmaloss = 0
 	for(var/obj/item/I in src)
-		if(I.contaminated)
+		if(I.contaminated && !(species.is_plasma_immune))
 			total_plasmaloss += zas_settings.Get(/datum/ZAS_Setting/CONTAMINATION_LOSS)
 		I.OnMobLife(src)
+	adjustToxLoss(total_plasmaloss)
+
 	if(status_flags & GODMODE)
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/human/life/handle_chemicals_in_body.dm]: This should be above the adjustToxLoss

**Review by Intigracy — DISMISSED** [review]

**Review by PJB3005 — CHANGES_REQUESTED** [review]
Make a new flags variable for this instead.

**Comment by BarneyGumball** [comment]
You should update the wiki to reflect this change: http://ss13.moe/wiki/index.php/Races#Plasma_Men

**Comment by jknpj** [comment]
<s>I liked being forever jitterry after getting shocked in a door

**Comment by 9600bauds** [comment]
Take coffee

**Comment by 9600bauds** [comment]
great, it already conflicts

**Comment by Intigracy** [comment]
You're not going to be able to get rid of the secret repo conflict, whoever merges this has to fix it.

code/modules/mob/new_player/new_player.dm

**Comment by Imagebard** [comment]
Maybe we should just remove the secret repo.

**Comment by PJB3005** [comment]
Hahaha haha a no

**Comment by ihadtoregisterforthis** [comment]
Plasmamen feeling hunger more like feature loss

**Review by PJB3005 — APPROVED** [review]

**Comment by 9600bauds** [comment]
Fucking hell I'm only gonna fix this up after I get a promise that someone will sekritrepo it up

**Comment by PJB3005** [comment]
@9600bauds make an account on http://git.nexisonline.net/ and I'll give you sekrit repo access.

**Comment by 9600bauds** [comment]
That'd be like

Sekrit repo selfmerging

which is logically the next step to selfmerging

**Comment by 9600bauds** [comment]
grepped through this conflict bait to see if I missed anything (flags don't give compile errors!!!!!!), it's OK

**Comment by Exxion** [comment]
Yeah that
That's already how it works, except worse

**Comment by Intigracy** [comment]
If someone merged it here then clearly you can just add it on the SR and blame their merge.

I remove the conflict tag from PRs after I fix them on the repo, and I fix them on the repo after they're merged.

**Comment by Intigracy** [comment]
Also fucking Imagebard I see that fucking loli image in my email convo for this PR.

**Comment by Exxion** [comment]
>Having e-mail notifications on

**Comment by Probe1** [comment]
Yeah github support offers their apologies but there's no un sending the email

**Comment by 9600bauds** [comment]
Someone PLEASE look at this conflict bait already

Even if there are no conflicts that Git detects, it has massive potential to cause future errors in other people's PRs
Say someone writes `species.flags & NO_SKIN` in their PR. That would be entirely incorrect after this flag refactor and wouldn't work as they intended - but Git wouldn't realize that, Travis wouldn't realize that, because IT DOESN'T THROW ANY ERRORS. They wouldn't even realize something was wrong unless they rebased to up-to-date and tested the same thing again.

**Review by ComicIronic — APPROVED** [review]
Nice flag split, looks consistent.

**Comment by ComicIronic** [comment]
Done.

**Comment by 9600bauds** [comment]
thank you Comic

needs sekrit repo magic to be merged

**Comment by PJB3005** [comment]
@9600bauds make that account on N3X15's GitLab server and tell me so I can give you secret repo access.

**Comment by 9600bauds** [comment]
This is the last time I fix conflicts on this thing
PLEASE do the needful for now, it's not my responsibility to have to self-merge it

**Comment by PJB3005** [comment]
What do you want me to make the changes to the secret repo?

**Comment by 9600bauds** [comment]
what do you do when anybody else without secret repo access gets secret repo conflicts

**Comment by PJB3005** [comment]
Well, do it myself.

Since you're, you know, collab, I thought you'd want to do it yourself.

**Comment by 9600bauds** [comment]
Oh, I didn't know the secret repo was an "every collaborator" thing, I never got that memo

Next time then
